frontend/src/components/application/AppVerificationControls/SetupControls.tsx

```diff
@@ -144,6 +145,9 @@ const AppVerificationSetup: FunctionComponent<Props> = ({ app }) => {
   } else {
     content = (
       <div className="space-y-3">
+        <p>{t("verification-instructions")}</p>
```

Comment:
Usually you don't want these to go too much into the width, as it get's hard to read.

I would recommend `xl:max-w-[75%]` at least. We do already use that in a few other places (or similar things)

For e.g. I like to use `prose dark:prose-invert` for text blocks, but that sometimes feels a bit to aggressive as it cuts down on the width hard. (probably for good reason)
https://tailwindcss.com/docs/typography-plugin#overriding-max-width

Reply:
How about

![image](https://user-images.githubusercontent.com/24497353/220722160-9f7413f2-0cb5-4339-9087-483aed36be80.png)